examples/complete-web/variables.tf

```diff
@@ -0,0 +1,28 @@
+locals {
+  env                  = var.env
```

Comment:
Please let's not use locals when there is no actual need in them (we're not constructing the variable). Unless you have a reason to keep them.
(here, everywhere in this file and other examples)